airgun/views/task.py

```diff
@@ -15,7 +15,7 @@ class TaskReadOnlyEntry(ReadOnlyEntry):
 class TasksView(BaseLoggedInView, SearchableViewMixin):
     title = Text("//h1[text()='Tasks']")
     table = SatTable(
```

Comment:
@pondrejk  As SatTable has been deprecated, please try to use Table instead

Reply:
Previous developers of Airgun decided `Table` has everything they need and started process of deprecating `SatTable`. They did not finish it, that's why you can find both in Airgun.

One important thing to note is return value for empty tables. `SatTable` returned empty list, `Table` returns cell content in all columns (usually something like "There were no search results"). See e.g. https://github.com/SatelliteQE/robottelo/pull/7212 for how this can be handled from Robottelo side.

Putting my Tier2 hat on, I appreciate Omkar pointing this out, but I do not consider it blocking. It's not the mess you have made and while it would be very nice from you to help clean it up, we can't require that.